### DISCUSSION TITLE
Are there plans to support multiple Rollout kinds for Statefulset, DaemonSet etc?

### DISCUSSION THREAD

**Discussion by alwaysastudent** [argoproj/argo-rollouts#1300] [Q&A]
`argo-rollouts` currently is limited to support only "Deployment" kind resources, which is great. But just was wondering if there are plans to support multiple Rollout kinds for Statefulset, DaemonSet etc.

**Comment by kostis-codefresh** [comment]
Already answered here https://github.com/argoproj/argo-rollouts/discussions/1346#discussioncomment-1197262

Also see https://github.com/argoproj/argo-rollouts/issues/1457
